packages/framework/src/Migrations/Version20240617204710.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20240617204710 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        if (!$this->isAppMigrationNotInstalledRemoveIfExists('Version20200624103557')) {
+            $this->sql('ALTER TABLE order_items DROP related_order_item_id');
```

Comment:
just a thought on WIP :innocent: I think we should provide a migration for those who actually use the relation on their project, what do you say?

Reply:
added, good idea :)